phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -631,7 +664,7 @@ private boolean addIndexesFromPhysicalTable(MetaDataMutationResult result, Long
         if (view.getType() != PTableType.VIEW || view.getViewType() == ViewType.MAPPED) {
             return false;
         }
-        String physicalName = view.getPhysicalName().getString();
+        String physicalName = view.getPhysicalNames().get(0).toString();
```

Comment:
Yes, we don't support views over multiple tables that's why I used get(0) only. But I have undo it by modifying the other api's to resolve schema and table-name correctly even if namespace separator is present. like SchemaUtil.getTableNameFromFullName()